sklearn/model_selection/_search.py

```diff
@@ -908,12 +912,10 @@ class GridSearchCV(BaseSearchCV):
            scoring=..., verbose=...)
     >>> sorted(clf.cv_results_.keys())
     ...                             # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
-    ['mean_fit_time', 'mean_score_time', 'mean_test_score',...
-     'mean_train_score', 'param_C', 'param_kernel', 'params',...
-     'rank_test_score', 'split0_test_score',...
-     'split0_train_score', 'split1_test_score', 'split1_train_score',...
-     'split2_test_score', 'split2_train_score',...
-     'std_fit_time', 'std_score_time', 'std_test_score', 'std_train_score'...]
+    ['mean_fit_time', 'mean_score_time', 'mean_test_score', 'param_C',...
```

Comment:
I don't get why this changed

Reply:
Because I am getting a doctest failure. This is because I change the default to none/false.

Reply:
Do you want `None` to behave like `True` till 0.22 ? If so, I will revert this change and will make other changes as well.